myhpom/static/myhpom/include/_footer.scss

```diff
@@ -1,48 +1,89 @@
 .footer-headline-enclosure {
     border-bottom: 1px solid rgba(0, 0, 0, 0.1);
     border-top: 1px solid rgba(0, 0, 0, 0.1);
+    background-color: $brand-blue;
 }
```

Comment:
In the prototype there is a white rule between the .footer-headline-enclosure and the .footer-lower-enclosure